sct-commons/src/test/java/org/lfenergy/compas/sct/commons/ExtRefServiceTest.java

```diff
@@ -214,12 +218,30 @@ void updateAllExtRefIedNames_when_FlowStatus_INACTIVE_should_remove_binding() {
         assertExtRefIsNotBound(extRef);
     }
 
+    private static Stream<Arguments> provideAllowedFcdaListEmptyOrNull() {
+        return Stream.of(
+                Arguments.of("Set of allowed FCDA is null", null),
+                Arguments.of("Set of allow FCDA is Empty", new HashSet<>())
```

Comment:
`new HashSet<>()` is fine, but you can also use `Set.of()` or `Collections.emptySet()` to be more explicit.

Reply:
ok